okhttp/src/main/java/feign/okhttp/OkHttpClient.java

```diff
@@ -106,6 +105,9 @@ static Request toOkHttpRequest(feign.Request input) {
 
   private static feign.Response.Body toBody(final ResponseBody input) throws IOException {
     if (input == null || input.contentLength() == 0) {
+      if (input != null) {
+        input.close();
```

Comment:
Hi Spencer!

It's apparently always okay to call close() on the body, even if you haven't read it. In this case, it should be doubly okay since we already checked that the content-length is 0 so there's nothing to read anyway.

We inserted the code here because on the very next line the toBody() method returns null, so callers have no way to access the unclosed ResponseBody object. OkHttp3 requires that every ResponseBody object is closed, regardless of whether or not the body has any contents.